server/api/mixed-content/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @fileoverview API Route
+ * Route for mixed content queries, it uses BigQuery's
+ * client and express's router.
+ *
+ * Queries are stored in /.queries.json, each has a name, description,
+ * sqlQuery and suggested visualizations.
+ *
+ */
+
+import {Router as router} from 'express';
+import {BigQuery} from '@google-cloud/bigquery';
+// This is a collection of all queries and their metadata in json.
+import * as queries from './queries.json';
+
+const mixedApi = router();
+const bigqueryClient = new BigQuery();
+
+mixedApi.get('/top-websites-with-mixed-content', async (req, res) => {
+  const query = await queries.TopWebsitesWithMixedContent;
```

Comment:
This is not a promise, so you can remove this `await` safely.
Same for the next queries.*

Reply:
Done!